src/CTA.Rules.PortCore/ProjectSpecificPort/WCF/WCFServicePort.cs

```diff
@@ -14,7 +14,7 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Constants = CTA.Rules.Update.WCF.Constants;
+using Constants = CTA.Rules.PortCore.WCF.Constants;
 
 namespace CTA.Rules.PortCore
 {
```

Comment:
Does this file belong in this project?

Reply:
This was moved to PortCore project and is only referenced in the WCFProjectRewriter which also lives in this project. Same as above with Github trying to display this in the diff view